packages/ndla-ui/src/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+import { Cross } from '@ndla/icons/action';
+import { useTranslation } from 'react-i18next';
+const CloseButtonWrapper = styled.div`
+  position: relative;
+  top: 1px;
+  right: 1px;
+`;
+
+const CloseBut = styled.button`
+  background-color: transparent;
+  border: none;
+  display: flex;
+`;
+
+type Props = {
+  onClick: () => void;
+  color: string;
+};
+
+export const CloseButton = ({ onClick, color }: Props) => {
+  const { t } = useTranslation();
+  return (
+    <CloseButtonWrapper aria-label={t('lukk')}>
```

Comment:
Det finnes ikke noen translation for `'lukk'`, men t-funksjonen defaulter tilbake til nøkkelen som sendes inn (i dette tilfellet returnerer den derfor `'lukk'`). Det finnes derimot en translation for `'close'`, se:
 https://github.com/NDLANO/frontend-packages/blob/5547de3650e346ec3414a5c5e54b669ac0f7c107/packages/ndla-ui/src/locale/messages-nb.ts#L789